contracts/IERC20.sol

```diff
@@ -1,30 +1,3 @@
 pragma solidity 0.6.6;
 
-
-interface IERC20 {
-    event Approval(address indexed _owner, address indexed _spender, uint256 _value);
-
-    function approve(address _spender, uint256 _value) external returns (bool success);
-
-    function transfer(address _to, uint256 _value) external returns (bool success);
-
-    function transferFrom(
-        address _from,
-        address _to,
-        uint256 _value
-    ) external returns (bool success);
-
-    function allowance(address _owner, address _spender) external view returns (uint256 remaining);
-
-    function balanceOf(address _owner) external view returns (uint256 balance);
-
-    function decimals() external view returns (uint8 digits);
-
-    function totalSupply() external view returns (uint256 supply);
-}
-
-
-// to support backward compatible contract name -- so function signature remains same
-abstract contract ERC20 is IERC20 {
-
-}
+import "@kyber.network/utils-sc/IERC20.sol";
```

Comment:
not sure what is the advantage for using the imported one?

Reply:
Not an advantage. I run into compilation issues if I don't use the imported one. Mainly how the IERC20 contract has already been defined.